<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageManipulationToolbox</class>
 <widget class="QWidget" name="ImageManipulationToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="horizontalGroupBox">
       <property name="title">
        <string>Convert selected image(s) to:</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_jpeg">
          <property name="text">
           <string>JPEG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_tif">
          <property name="text">
           <string>TIF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_png">
          <property name="text">
           <string>PNG</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
